lib/packages/mkdir.rb

```diff
@@ -0,0 +1,12 @@
+require "set"
```

Comment:
I guess we'll start following this rule from now on. single quotes if interpolation isn't required